<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZWOSettings</class>
 <widget class="QFrame" name="ZWOSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="spinbox_gridlayout">
     <item row="2" column="0">
      <widget class="QLabel" name="red_label">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gamma_label">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="gamma_spinbox"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="blue_spinbox"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="red_spinbox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="blue_label">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="temperature_label">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="temperature_spinbox"/>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="usb_spinbox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="usb_label">
       <property name="text">
        <string>USB Bandwidth</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="checkbox_gridlayout">
     <item row="2" column="0">
      <widget class="QCheckBox" name="horizontalflip_checkbox">
       <property name="text">
        <string>Horizontal Flip</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="hardwarebin_checkbox">
       <property name="text">
        <string>Bin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="highspeed_checkbox">
       <property name="text">
        <string>High Speed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="checkbox_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="verticalflip_checkbox">
       <property name="text">
        <string>Vertical Flip</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
